dotcom-rendering/src/components/Standfirst.tsx

```diff
@@ -310,6 +316,19 @@ const standfirstStyles = ({ display, design, theme }: ArticleFormat) => {
 					return css`
 						color: ${palette('--standfirst-text')};
 					`;
+				case ArticleDesign.Gallery:
+					return css`
+						${grid.between('centre-column-start', 'grid-end')}
+						color: ${palette('--standfirst-text')};
+						background-color: ${palette('--standfirst-background')};
+						max-width: 280px;
+						${from.tablet} {
+							max-width: 460px;
+						}
```

Comment:
The max-width is how frontend was limiting the width of the standfirst. In AR, it is not limiting the width so the standfirst can go all the way through the center collumn as shown here:

![image](https://github.com/user-attachments/assets/0e855158-54d2-4248-b158-631e32f8db38)

Maybe this is how we want it to look like in the web too?